### PR TITLE
Checkpoint flexibility

### PR DESCRIPTION
**Pull request by msamogh**
**Proposed changes**:
- Mention that checkpoints can also start with bot actions

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: examples/moodbot/domain.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ actions:
 
 templates:
   utter_greet:
-  - text: "Hey! How are you?"
+  - text: "Heyeesf! How are you?"
```
Comment (tmbo) [inline examples/moodbot/domain.yml]: I guess these changes are not intended?

Comment (msamogh) [reply examples/moodbot/domain.yml]: Oops! Fixed it.

**Comment by erohmensing** [comment]
@tmbo 1.2.x? 😄

**Comment by tmbo** [comment]
yes

**Comment by erohmensing** [comment]
ok turns out trying to change base that way did _not_ work as expected hahaha

**Pull request merged**